main-unix.c

```diff
@@ -188,6 +188,36 @@ assert_handler (const jerry_api_object_t *function_obj_p __attr_unused___, /** <
   }
 } /* assert_handler */
 
+static void
+print_usage (char **argv)
```

Comment:
It is smelling a little bit bad to have an array passed without knowing its size. Especially since we are only using its 0-th element. Why not just have `char *name` as an argument?